core-feature-pack/common/src/main/resources/content/bin/domain.conf.ps1

```diff
@@ -89,3 +89,6 @@ if ($HOST_CONTROLLER_JAVA_OPTS -eq $null) {
 # Sample JPDA settings for shared memory debugging
 #PROCESS_CONTROLLER_JAVA_OPTS="$PROCESS_CONTROLLER_JAVA_OPTS -agentlib:jdwp=transport=dt_shmem,server=y,suspend=n,address=jboss"
 #HOST_CONTROLLER_JAVA_OPTS="$HOST_CONTROLLER_JAVA_OPTS -agentlib:jdwp=transport=dt_shmem,server=y,suspend=n,address=jboss"
```

Comment:
Not connected to this PR, but should those assignments start with `$sign` (e.g. `$HOST_CONTROLLER_JAVA_OPTS`)?

Reply:
Good catch!, variables in Powershell are defined starting the with $, I'll append a commit